src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -313,16 +318,18 @@ def build(self) -> SlackBody:
         event_for_tags = self.event or self.group.get_latest_event()
         color = get_color(event_for_tags)
         fields = build_tag_fields(event_for_tags, self.tags)
-        footer = build_footer(self.group, self.issue_alert, project, self.rules)
+        footer = (
+            build_notification_footer(self.notification, self.recipient)
+            if self.notification and self.recipient
+            else build_footer(self.group, project, self.rules)
+        )
         obj = self.event if self.event is not None else self.group
-
-        if not self.issue_alert:
+        if not self.issue_details or self.recipient and isinstance(self.recipient, Team):
```

Comment:
Can you add parenthesis to the conditional? Also `isinstance(None, X)` is always false so you be able to simplify the condition.